helm-prereqs/setup.sh

```
@@ -755,51 +844,117 @@ echo "=== [4/6] unseal vault ==="
+            _db_ready=true
+            break
+        fi
+        echo "  database ${_db_name} is still being reconciled (${_db_attempt}/12) — retrying in 5s..."
+        sleep 5
+    done
+    if [[ "${_db_ready}" != "true" ]]; then
+        echo "ERROR: failed to create PostgreSQL database ${_db_name} owned by ${_db_owner}" >&2
+        exit 1
+    fi
+done < <(kubectl get postgresql nico-pg-cluster -n postgres \
+    -o go-template='{{range $database, $owner := .spec.databases}}{{$database}}={{$owner}}{{"\n"}}{{end}}')
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)